src/fidesops/models/privacy_request.py

```diff
@@ -33,9 +55,20 @@ class PrivacyRequestStatus(EnumType):
     in_processing = "in_processing"
     pending = "pending"
     complete = "complete"
+    paused = "paused"
     error = "error"
 
 
+def generate_request_callback_jwe(webhook: PolicyPreWebhook) -> str:
+    """Generate a JWE to be used to resume privacy request execution."""
+    jwe = WebhookJWE(
+        webhook_id=webhook.id,
+        scopes=[PRIVACY_REQUEST_CALLBACK_RESUME],
+        iat=datetime.now().isoformat(),
```

Comment:
Sean and I chatted about this - we're including the scopes in the token we give to the user to follow the same format as our other oauth endpoints.  However, note that we gave these scopes in the token to the user, and they are just handing it back to us.